gostream/cmd/stream_av/main.go

```diff
@@ -128,8 +130,9 @@ func runServer(
 		err = multierr.Combine(err, <-audioErr, server.Stop(ctx))
 	}()
 
+	log := logging.NewLogger("stream_av")
```

Comment:
We don't use these files very much, so I just added in a quick fix for the new logger situation.